projects/models.py

```diff
@@ -235,16 +238,22 @@ class SectionTypes(models.TextChoices):
         db_index=True,
     )
     override_permissions = models.ManyToManyField(User, through="ProjectUserPermission")
-    # Not discoverable, but can still be accessed via the URL
-    unlisted = BooleanField(default=False, db_index=True)
+
+    visibility = models.CharField(
+        choices=Visibility.choices,
+        default=Visibility.NOT_IN_MAIN_FEED,
+        db_index=True,
+    )
 
     # Whether we should display tournament on the homepage
     show_on_homepage = models.BooleanField(default=False, db_index=True)
 
     objects = models.Manager.from_queryset(ProjectsQuerySet)()
 
     # Annotated fields
-    followers_count = models.PositiveIntegerField(default=0, db_index=True, editable=False)
+    followers_count = models.PositiveIntegerField(
+        default=0, db_index=True, editable=False
+    )
```

Comment:
I don't have an objection to this, but seems out of scope for this PR

Reply:
It was just autoformatting, no actual changes here

Reply:
Oh, haha I should have checked more closely. no problem